examples/plot_pvlaues.py

```diff
@@ -0,0 +1,193 @@
+    t_val = beta / stderr
+    # and p-values
+    p_val = 2 * stats.t.sf(np.abs(t_val), dfs)
+
+    # project results back to channels x time points space
+    beta = beta.reshape((n_channels, n_times))
+    stderr = stderr.reshape((n_channels, n_times))
+    t_val = t_val.reshape((n_channels, n_times))
+    # replace p-values == 0 with asymptotic value `tiny`
+    p_val = np.clip(p_val, tiny, 1.).reshape((n_channels, n_times))
+
+    # create evoked object for plotting
+    lm_betas[predictor] = EvokedArray(beta, epochs_info, tmin)
+    stderrs[predictor] = EvokedArray(stderr, epochs_info, tmin)
+    t_vals[predictor] = EvokedArray(t_val, epochs_info, tmin)
+    p_vals[predictor] = p_val
+    # transform p-values to Shannon information values (i.e., surprise values)
+    s_vals[predictor] = EvokedArray(-np.log2(p_val)*1e-6, epochs_info, tmin)
+
+
+###############################################################################
+# --- plot inference results for predictor = 'phase-coherence' ---
+
+predictor = 'phase-coherence'
+
+# only show -250 to 500 ms
+ts_args = dict(xlim=(-.25, 0.5),
+               # use unit=False to avoid conversion to micro-volt
+               unit=False)
+topomap_args = dict(cmap='RdBu_r',
+                    # keep values scale
+                    scalings=dict(eeg=1),
+                    average=0.05)
+# plot t-values
+fig = t_vals[predictor].plot_joint(ts_args=ts_args,
+                                   topomap_args=topomap_args,
+                                   title='T-values for predictor %s'
+                                         % predictor,
+                                   times=[.13, .23])
+fig.axes[0].set_ylabel('T-value')
+
+###############################################################################
+# correct p-values for multiple testing and create a mask for non-significant
+# time point dor each channel.
+reject_H0, fdr_pvals = fdr_correction(p_vals[predictor],
+                                      alpha=0.01)
+# plot t-values, masking non-significant time points.
+fig = t_vals['phase-coherence'].plot_image(time_unit='s',
+                                           mask=reject_H0,
+                                           unit=False,
+                                           # keep values scale
+                                           scalings=dict(eeg=1))
+fig.axes[1].set_title('T-value')
+
+###############################################################################
+# plot surprise-values as "erp"
+# only show electrode `B8`
+pick = limo_epochs['2'].info['ch_names'].index('B8')
+fig, ax = plt.subplots(figsize=(7, 4))
+ax = plot_compare_evokeds(s_vals[predictor],
+                          picks=pick,
+                          axes=ax,
+                          show_sensors='upper left')
+plt.rcParams.update({'mathtext.default':  'regular'})
+ax.axes[0].set_ylabel('$S_{value}$ (-$log_2$($P_{value}$)')
+ax.axes[0].yaxis.set_label_coords(-0.1, 0.5)
+plt.plot()
```

Comment:
@JoseAlanis does this now give the nicer looking results, consistent with R?

Reply:
Yes, the code is slightly different to the one implemented in mne.stats.linear_regression and gives us results consistent with R'w lm() (more specifically passing the summary()? or anova()` command to the lm object)